Slider/themes/Slider_template.less

```diff
@@ -1,134 +1,182 @@
 	vertical-align: middle;
-	border-collapse: separate;
 	cursor: pointer;
-	padding: @d-slider-halo-size; // halo to make touches easier
-	.user-select(none); // prevent selection
+	padding: @d-slider-halo-size;
+	.user-select(none);
 	.box-sizing(border-box);
-	.tap-highlight-color(rgba(255,255,255,0));
+	.tap-highlight-color(rgba(255, 255, 255, 0));
+
 	input {
-		display: none;
+	display: none;
+	}
+
+	//
+	// d-slider-bar
+	//
+	& .d-slider-bar {
+	border: @d-slider-bar-border-width @d-slider-bar-border-style @d-slider-bar-border-color;
+	border-radius: @d-slider-bar-border-radius;
+	padding: 0;
+	margin: 0;
+	background-clip: content-box;
+	background-color: @d-slider-bar-color;
+	}
+	//
+	// d-slider-container
+	// this is the bar below the slider handle(s)
+	//
+	& .d-slider-container {
+	.box-sizing(border-box);
+	position: relative;
+	height: 100%;
+	width: 100%;
+	box-shadow: 0 0.1em 0.2em 0.1em rgba(0, 0, 0, 0.1) inset;
+	}
+
+	//
+	// d-slider-progress-bar
+	// this is the bar that start from min, up to the slider handle
+	// when the slider has 2 values, this is the bar between both values
+	//
+	& .d-slider-progress-bar {
+	.box-sizing(content-box);
+	border-width: 0;
+	background-color: lighten(#000000, 60%);
+	}
+
+	&.d-disabled {
+	cursor: default;
+	.box-shadow(none);
+	}
+	&.d-disabled .d-slider-container {
+	.box-shadow(none);
+	border-color: lighten(@d-slider-bar-color, 20%);
+	}
+	&.d-disabled .d-slider-progress-bar {
+	background: none;
+	}
+	&.d-disabled .d-slider-handle {
+	&, &:hover, &:focus, &:active, &.active {
+		border-color: lighten(@d-slider-handle-border-color, 20%);
+		.box-shadow(none);
+		background-color: @d-slider-handle-color;
+	}
+	}
+}
+
+.d-slider-container > * {
+	position: absolute;
+}
+
+//
+// d-slider-handle
+//
+.d-slider-handle {
+	.user-select(none);
+	position: absolute;
+	.box-sizing(border-box);
+	border: @d-slider-handle-border-width @d-slider-handle-border-style @d-slider-handle-border-color;
+	border-radius: @d-slider-handle-border-radius;
+	padding: 0;
+	margin: 0;
+	background-clip: content-box;
+	background-color: @d-slider-handle-color;
+	z-index: 1;
+	&, &:active, &.active {
+	&:focus {
+		outline: thin dotted;
+		outline-offset: -3px;
+	}
+	}
+	&:hover, &:focus {
+	color: @btn-default-color;
+	text-decoration: none;
+	}
+	&:focus + .d-slider-handle {
+	z-index: 0; // Move down the second handle if the first is focused
+	}
+	&:active, &.active {
+	outline: 0;
+	background-image: none;
+	.box-shadow(inset 0 3px 5px rgba(0, 0, 0, .125));
 	}
-	.d-slider-styles;
 }
+
+// =========================================
+// rules below only affect size/position
+// to handle horizontal/vertical/flip
+// =========================================
+
+//
+// horizontal slider
+//
 .d-slider-h {
 	width: @d-slider-size;
 	.d-slider-container {
-		width: @d-slider-bar-size;
-		min-width: (@d-slider-knob-size*4);
-		height: @d-slider-bar-thickness;
+	min-width: (@d-slider-handle-size * 4);
```

Comment:
don't you have tabbing/spacing issues here? (or was the issue in the original file?) (same at other places you don't seem to intend after {)